src/utils/ETO.js

```diff
@@ -0,0 +1,224 @@
+import { computeICOTransparency, getICOLogs } from '../utils';
+import {
+  allocateCSVFile,
+  drawStatistics,
+  hideLoader,
+  setEtoProperties,
+  setSmartContractLoaded, showIcoNotStarted, showLoader, showStatistics,
+} from '../actions/ScanAction';
+import {
+  getAbiAsDictionary, getERC20Parameters,
+  getSmartContract,
+} from './web3';
+import { showErrorMessage } from '../actions/ModalAction';
+import { getStatistics, initStatistics } from './stats';
+import {
+  setConversionRate,
+  setStatisticsByCurrency,
+} from '../actions/CurrencyAction';
+
+
+export const getETOTokenSmartContract = (web3, etoConfig, isCrowdSale) => {
+  if (!web3) { return null; }
+  let address;
```

Comment:
We can use ternary operator isntead of if...else here

Reply:
I think ternary operators are really handy for assignment operations. As here it's multiple lines inside if statement it's better to keep if-else. What do you think?

Reply:
I'm fine with that 👍